presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeExecution.java

```diff
@@ -66,8 +66,6 @@ public void testJoins()
     @Test
     public void testFailures()
     {
-        assertQueryFails("SELECT sequence(1, orderkey) FROM orders",
```

Comment:
This test is verifying the behavior when function doesn't exist. Instead of removing, would you change the test to use a different function?